api/async.rb

```diff
@@ -15,8 +15,12 @@
 require 'api/timeout'
 
 module Api
-  # Represents an asynchronous operation definition
+  # Base class from which other Async classes can inherit.
   class Async < Api::Object
```

Comment:
Can you move the shared fields up into this definition?

Reply:
I think this is a good idea but I'm worried about the size of this PR at this point. I'll add it in a follow on PR

Reply:
It should be the same size, since they won't need to go in PollAsync, right?

Reply:
Moving timeouts to the inherited class would mean moving them up out of the async.operation block. This would be another change across resource.erb, website generation templates and every api.yaml.

Reply:
ah yeah they don't really belong in that block anyways. sg